x-pack/plugins/license_management/server/index.ts

```diff
@@ -17,4 +17,5 @@ export const config: PluginConfigDescriptor<LicenseManagementConfig> = {
   exposeToBrowser: {
     ui: true,
   },
+  deprecations: ({ deprecate }) => [deprecate('enabled', '8.0.0')],
```

Comment:
License Management's UI can be disabled via `licenseManagement.ui.enabled: false`. Is there a way to guide users to use this setting instead?